Sources/FronteggSwift/models/plist/FronteggPlist.swift

```diff
@@ -65,8 +67,18 @@ struct FronteggPlist: Decodable, Equatable {
 
         let logLevel = try container.decodeIfPresent(LogLevel.self, forKey: .logLevel)
         self.logLevel = logLevel ?? .warn
-
-        self.payload = try Payload(from: decoder)
+        
+        do {
+            self.payload = try Payload(from: decoder)
+        } catch {
+            if lateInit != true {
+                throw error
+            }
+            
+            let emptyPayload = try JSONSerialization.data(withJSONObject: ["baseUrl":"", "clientId":""], options: [])
+            let defaultLateInitPayload = try JSONDecoder().decode(Payload.self, from: emptyPayload)
```

Comment:
What happens if TRY will fail

Reply:
`try` will not fail here. This try is for decoding a serialized JSON, and since it's a `constant`, it will **NOT** throw an exception.